presto-main/src/test/java/io/prestosql/execution/buffer/BenchmarkBlockSerde.java

```diff
@@ -187,30 +292,20 @@ public void setup(Iterator<Page> pagesIterator)
             dataSource = sliceOutput.slice();
         }
 
-        public void setup(Type type, Iterator<?> values)
+        public <T> void setup(Type type, Iterator<T> values, BiConsumer<BlockBuilder, T> writer)
                 throws Exception
         {
             pagesSerde = new TestingPagesSerdeFactory(new TestingBlockEncodingSerde(), false).createPagesSerde();
```

Comment:
Not introduced in this PR, but this should use the regular `PagesSerdeFactory` instead of `TestingPagesSerdeFactory` since the testing version produces a `PagesSerde` that includes extra synchronization that might be material in the benchmark results.

Reply:
Curious, do you happen to know why the testing one is more synchronized?

Reply:
I assume that it’s because PagesSerDe is not thread safe and adding synchronized to the methods was deemed easier to use in multi threaded  tests.

Reply:
@pettyjamesm we don't use synchronized factory here. Synchronized factory is only used when `io.prestosql.execution.buffer.TestingPagesSerdeFactory#testingPagesSerde` is used. It's not used here